### ISSUE TITLE
Change password dont lose login

### ISSUE THREAD

**Issue by rube200** [pi-hole/web#2534]
### Versions

- Pi-hole:  [v5.15.5](https://github.com/pi-hole/pi-hole/releases/v5.15.5)
- AdminLTE:  [v5.18.4](https://github.com/pi-hole/AdminLTE/releases/v5.18.4)
- FTL:  [v5.21](https://github.com/pi-hole/FTL/releases/v5.21)

### Platform

- OS and version:  Debian 11

- Platform:  Raspberry Pi

### Expected behavior
So i was messing with password and tryied to change it using the command `pihole -a -p test` and went back to the pihole and still had login.

### Actual behavior / bug

After change password login should be lost to avoid security issues.

**Comment by dschaper** [comment]
I don't know of too many other systems that invalidate existing sessions on password change. In fact most systems ask if you want to log out existing sessions specifically and separately from password changes.

**Comment by dschaper** [comment]
But this is a Feature Request and not a bug report so:

Because of the number of requests and the very limited resources we have as a free open-source project run by volunteers, we ask that you open all Feature Requests at our [Discourse Forum](https://discourse.pi-hole.net/c/feature-requests/8). 

Thank you for your understanding.

**Comment by rube200** [comment]
I mean systems like google and discord does that.
For example if some reason the password gets compromise it will be a gate after changing the password until it expires.

**Comment by yubiuser** [comment]
Fixed in v6

https://github.com/pi-hole/FTL/commit/5e436312f71ea73ea7f47dbbab0d489301b0147f

**Comment by SodaWithoutSparkles** [comment]
Should issues that are WIP be considered complete/stale?

**Comment by yubiuser** [comment]
They probably should not be marked as stale...

https://github.com/pi-hole/AdminLTE/blob/d7f1842825e33abe098a9383672b63d6d56dfd0f/.github/workflows/stale.yml#L23

**Comment by SodaWithoutSparkles** [comment]
@yubiuser `WIP` is not in that list however. Should that be a PR?

------
Edit:
Also, perhaps that CRON job should run more frequently. It is confusing to see that label don't get removed in a few minites after a comment.

Should the discussion about that github action be on a separate bug report? Just to move unrelated discussion off this issue and not tagging unrelated people.

**Comment by yubiuser** [comment]
> Should that be a PR?

No, just a hint which labels are exempt. But I'm planing to change that anyway ... we agreed to unify the lables used across all repos.

> Also, perhaps that CRON job should run more frequently. It is confusing to see that label don't get removed in a few minites after a comment

You're right. The workflow only triggers once a day at 08:00. A good value might be something between every 15 and 60 minutes. Do you want to PR this?

**Comment by yubiuser** [comment]
>  A good value might be something between every 15 and 60 minutes.

Or even better: keep the run at 08:00 and add https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#issue_comment

**Comment by SodaWithoutSparkles** [comment]
Nah, I am not near a computer. You start a PR?

**Comment by SodaWithoutSparkles** [comment]
@yubiuser It seems that WIP issues still get marked as stale

**Comment by SodaWithoutSparkles** [comment]
Nope

**Comment by yubiuser** [comment]
The issue should be finally completely fixed by https://github.com/pi-hole/FTL/pull/1630 in FTL v6

**Comment by yubiuser** [comment]
Pi-hole v6 has been released
